_episodes/25-section2-optional-exercises.md

```diff
@@ -0,0 +1,44 @@
+---
+title: "Optional exercises for section 2"
+start: false
+teaching: 0
+exercises: 45
+questions:
+- "What is a desirable way to measure and use test coverage?"
+objectives:
+- "Further explore how to measure and use test coverage."
+keypoints:
+---
+
+This episode holds some optional exercises for section 2. 
+The exercises have an explorative nature, so see what you like to explore.
+For the remainder of the lesson we recommend sticking with the tools that were introduced in this section.
+
+> ## Exercise: Apply to your own project(s)
+> Apply what you learned in this section to your own project(s). 
+> This is the time to ask any questions to our instructors or let us help you solve problems that you run into.
```

Comment:
As with the optional exercise in Section 1, probably good to give just a bit more guidance, e.g. trying adding some unit tests with pytest to an existing project, add CI to an existing project that runs these tests, think of some edge cases that you might not have tested, etc.

Reply:
Yes, good one! Added with 1c007f18095c24d318ac0c8